Native/CommandPaletteNucleoFFI/src/lib.rs

```
@@ -482,6 +486,57 @@ fn keyword_exact_line_score(query_char_count: usize) -> f64 {
     1_800.0 + f64::from(query_char_count as u32) * 10.0
 }
 
+fn title_phrase_score(candidate: &Candidate, query: &str, query_token_count: usize) -> Option<f64> {
+    if query.is_empty() {
+        return None;
+    }
+
+    let title = candidate.title.to_lowercase();
+    let query = query.to_lowercase();
+    let Some(start_byte) = title.find(&query) else {
```

Clippy (1.95.0)

[warning] this `let...else` may be rewritten with the `?` operator

(warning)